sql/executor.go

```diff
@@ -99,6 +99,11 @@ func (e *Executor) Execute(args driver.Request) (driver.Response, int, error) {
 		}
 		planMaker.setTxn(txn, planMaker.session.Txn.Timestamp.GoTime())
 	}
+	var err error
+	planMaker.evalCtx.Location, err = getTimeZone(planMaker.session)
```

Comment:
Didn't mean to ignore your suggestion, I hadn't followed it. I'm going to leave it as is because I worry about a single roachnode having a bad zone file, and with the current active evaluation, the client will receive a http server internal error which is nice.